httpref_test.go

```diff
@@ -86,3 +87,13 @@ func TestReferences_Titles(t *testing.T) {
 	n := Statuses.Titles()
 	assert.Equal(t, 5, len(n))
 }
+
```

Comment:
Add test case to ensure that both WellKnownPort and RegisteredPorts do not include funny characters. Valid ones:

* 8080
* 8080-8081

Reply:
(Note to my future self: always select a range of code relevant to the comment)